main.go

```diff
@@ -52,6 +55,11 @@ func main() {
 	)
 	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
 
+	if err != nil {
+		log.Fatal(err)
```

Comment:
What will happen in log.Fatal() func?  

You can read about os.Exit(1) 

https://go.dev/play/p/pwfwPbQ8nnb